content/blog/2019-08-15-new-react-devtools.md

```diff
@@ -14,7 +14,7 @@ It also offers full support for React Hooks, including inspecting nested objects
 
 [Visit the interactive tutorial](https://react-devtools-tutorial.now.sh/) to try out the new version or [see the changelog](https://github.com/facebook/react/blob/master/packages/react-devtools/CHANGELOG.md#400-august-15-2019) for demo videos and more details.
```

Comment:
`Visit the interactive tutorial` translate this to hindi
`see the changelog` translate this to hindi